gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -120,6 +123,8 @@
   private static final StorageResourceId RESOURCE_ID =
       new StorageResourceId(BUCKET_NAME, OBJECT_NAME);
 
+  private static final int HTTP_PERMANENT_REDIRECT = 308;
```

Comment:
should we name the field `HTTP_RESUME_INCOMPLETE` ?

If we are ok with using `HTTP_PERMANENT_REDIRECT`, we could re-use the field `HttpStatusCodes.STATUS_CODE_PERMANENT_REDIRECT`

Reply:
`HttpStatusCodes.STATUS_CODE_PERMANENT_REDIRECT` is private so we can not use it. I named field `HTTP_PERMANENT_REDIRECT` because this is HTTP code name, but apparently GCS redefined it: https://cloud.google.com/storage/docs/json_api/v1/status-codes#307_Temporary_Redirect

Renamed to `STATUS_CODE_RESUME_INCOMPLETE`.